pkg/util/versions/versions.go

```diff
@@ -0,0 +1,31 @@
+package versions
+
+import (
+	"fmt"
+
+	"github.com/blang/semver"
+)
+
+// CalculateFeatureCompatibilityVersion returns a version in the format of "x.y"
+func CalculateFeatureCompatibilityVersion(versionStr string) string {
+	v1, err := semver.Make(versionStr)
+	if err != nil {
+		return ""
+	}
+
+	baseVersion, _ := semver.Make("3.4.0")
+	if v1.GTE(baseVersion) {
+		ans, _ := MajorMinorVersion(versionStr)
+		return ans
```

Comment:
[q] is this `MajorMinorVersion` function used anywhere else (since it's exported)?

if not, I think we can get rid of it. In this case all it does is recreating the exact same semver type that here is `v1`, we can just replace these two lines with:

`	 return fmt.Sprintf("%d.%d", v1.Major, v1.Minor)
`

Reply:
ah you're right, this was taken from enterprise, but for now we don't have a different use for it 👍